.evergreen/linker-tests.sh

```diff
@@ -77,18 +77,13 @@ $CMAKE --build . --target install --config RelWithDebInfo
 cd ..
 git reset --hard
 git apply --ignore-whitespace "$(system_path $linker_tests_deps_root/bson_patches/libbson2.patch)"
-cd cmake-build
-INSTALL_PATH="$(system_path $linker_tests_root/install/bson2)"
-SRC_PATH="$(system_path ../)"
-$CMAKE -DENABLE_MONGOC=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo $ADDITIONAL_CMAKE_FLAGS -DCMAKE_INSTALL_PREFIX="$INSTALL_PATH" "$SRC_PATH"
-$CMAKE --build . --target install --config RelWithDebInfo
+LIBBSON2_SRC_DIR="$(system_path "$PWD")"
 
 # Build libmongocrypt, static linking against libbson2
 cd $linker_tests_root/libmongocrypt-cmake-build
-PREFIX_PATH="$(system_path $linker_tests_root/install/bson2)"
 INSTALL_PATH="$(system_path $linker_tests_root/install/libmongocrypt)"
 SRC_PATH="$(system_path $libmongocrypt_root)"
-$CMAKE -DCMAKE_BUILD_TYPE=RelWithDebInfo $ADDITIONAL_CMAKE_FLAGS -DCMAKE_PREFIX_PATH="$PREFIX_PATH" -DCMAKE_INSTALL_PREFIX="$INSTALL_PATH" "$SRC_PATH"
+$CMAKE -DCMAKE_BUILD_TYPE=RelWithDebInfo "-DMONGOCRYPT_MONGOC_DIR=$LIBBSON2_SRC_DIR" $ADDITIONAL_CMAKE_FLAGS -DCMAKE_INSTALL_PREFIX="$INSTALL_PATH" "$SRC_PATH"
 $CMAKE --build . --target install --config RelWithDebInfo
```

Comment:
We no longer need to install the libbson2 with the patch, we just need to build libmongocrypt and point it at the patched version of mongo-c-driver.

Reply:
Nice